src/etc/inc/gwlb.inc

```diff
@@ -993,3 +979,22 @@ function running_dpinger_processes()
 
     return $result;
 }
+
```

Comment:
better to move `get_gateway_error()` to the monit script itself, it's not used elsewhere.